modules/appeals_api/app/workers/appeals_api/notice_of_disagreement_pdf_submit_job.rb

```diff
@@ -14,13 +14,13 @@ class NoticeOfDisagreementPdfSubmitJob
     include Sidekiq::MonitoredWorker
     include CentralMail::Utilities
 
-    def perform(id, retries = 0)
+    def perform(id, retries = 0, version = 'V1')
```

Comment:
I thought we got rid of the retries already? Or was that just for the NOD submit job? :\

Reply:
Can you guys help me remember why we removed retries from HLR but not NOD? Is that something we're going to do but haven't yet, or is there a reason for one and not the other?

Reply:
nm found the original ticket for the HLR retries.